frontend/src/stories/DropdownMenu.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
+
+<Meta title="Patterns/Dropdown menu" />
+
+# Dropdown menu
+
+> Example: A dropdown menu with four items
+
+The dropdown menu is a contextual menu triggered by clicking on a dropdown button. It provides options to the use.
+
+## Structure and Options
+
+The dropdown menu consists of a list of items, generally *menu rows*, with an optional title.
+
+The title is not interactive. It serves to provide additional context, if necessary.
+
+A menu item is interactive and generally represents an action. 
```

Comment:
Should we add here that the menu rows might content an optional icon?

Reply:
Good point! Added.